### PR TITLE
NATIVE-401: Refactor Term Extra to Offer

### PR DESCRIPTION
**Pull request by Taggotty**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: locales/locales.json [hunk 1]
```diff
@@ -2059,7 +2057,46 @@
   },
   "search": {
     "de": {
-      "searchPlaceholder": "Suche"
+      "searchCategory": "Suche nach einer Kategorie"
```
Comment (klinzo) [inline locales/locales.json:2060]: As we call categories now `Local Information`, this is probably wrong. Not sure and also not part of your task, so feel free to ignore this.

File: locales/locales.json [hunk 2]
```diff
@@ -1692,8 +1692,7 @@
       "noPermission": "Standortzugriff erforderlich",
       "loading": "Bestimme Standort...",
       "notAvailable": "Standortbestimmung deaktiviert",
-      "timeout": "Standortbestimmung fehlgeschlagen",
-      "refresh": "Liste der Orte aktualisieren"
```
Comment (klinzo) [inline locales/locales.json:1696]: Why has this been removed? Is this not needed anymore?

File: src/modules/endpoint/DatabaseConnector.js [hunk 3]
```diff
@@ -373,7 +373,7 @@ class DatabaseConnector {
         code: jsonObject.code,
         live: jsonObject.live,
         eventsEnabled: jsonObject.events_enabled,
-        extrasEnabled: jsonObject.extras_enabled,
+        offersEnabled: jsonObject.extras_enabled,
```
Comment (klinzo) [inline src/modules/endpoint/DatabaseConnector.js:376]: This has to stay for backwards compatibility, right?

Comment (Taggotty) [reply src/modules/endpoint/DatabaseConnector.js:376]: Hmm. I am not very familiar with the database files in the app, but I would geuess this belongs to [NATIVE-549](https://issues.integreat-app.de/browse/NATIVE-549)

File: locales/locales.json (hunk 2 above)
Comment (Taggotty) [reply locales/locales.json]: The locales branch was behind its master. Fixed it.

**Comment by maxammann** [comment]
I created a task to change extras in the DatabaseConnector: https://issues.integreat-app.de/browse/NATIVE-569

**Pull request merged**